webapp/src/components/sidebar_right/sidebar_right.jsx

```diff
@@ -122,15 +124,15 @@ export default class SidebarRight extends React.PureComponent {
     }
 
     getInIssues() {
```

Comment:
```suggestion
    getInIssuesCount() {
```

It looks like these 3 functions aren't actually called though. I think we can just remove them